docs/rfcs/000-template.md

```diff
@@ -41,3 +41,9 @@ Why should we *not* do this?
 
 - Can this functionality be introduced in multiple, distinct, self-contained pull requests?
 - A specification for when the feature is considered "done."
+
+## Feature description for Atom release blog post
+
+- When this feature is shipped, what would we like to say or show in our Atom release blog post (example: http://blog.atom.io/2018/07/31/atom-1-29.html)
```

Comment:
Is this for the blog?  Or for the release notes?  I didn't get clarity on that during the mini summit discussion.  I think targeting this towards release notes might be more useful, because:
-  not every feature is going to make it into the blog post
- the hard part about a release is figuring out if a specific pull request belongs in the "notable changes" section

Reply:
I was imagining this would be for the blog. If I remember correctly the mini summit discussion was around how the release process can take forever and part of that pain was creating a blog post from nothing. The person who is shipping and writing the blog post has to do a bunch of research in order to figure out what to include, and often ends up asking feature authors to provide blurbs because they're the ones in the best position to speak about the work they did. We can help streamline the process for ourselves and the release coordinator by writing post blurbs right when we finish the feature. We can even copy/paste it over to the release post draft immediately after merging. That might be a good thing to suggest as the final bullet point in this section...

And I think for most new features worthy of writing an RFC there's probably a good chance they deserve a mention in the release blog post. And it would certainly deserve a spot in the "notable changes" section of the release notes.

That's my 2 cents. Open to other suggestions.